src/layout/SigneeList/api.ts

```diff
@@ -0,0 +1,33 @@
+import { queryOptions } from '@tanstack/react-query';
+import { z } from 'zod';
+
+import { httpGet } from 'src/utils/network/sharedNetworking';
+import { appPath } from 'src/utils/urls/appUrlHelper';
+
+const signeeStateSchema = z
+  .object({
+    name: z.string().nullish(),
+    organisation: z.string().nullish(),
+    hasSigned: z.boolean(),
+    delegationSuccessful: z.boolean(),
+    notificationSuccessful: z.boolean(),
+  })
+  .refine(({ name, organisation }) => name || organisation, 'Either name or organisation must be present.');
+
+export type SigneeState = z.infer<typeof signeeStateSchema>;
+
+export const signeeListQuery = (partyId: string, instanceGuid: string) =>
+  queryOptions({
+    queryKey: ['signeeList', partyId, instanceGuid],
+    queryFn: () => fetchSigneeList(partyId, instanceGuid),
+    staleTime: 1000 * 60 * 10, // 10 minutes
```

Comment:
Is it important to set this explicitly? The default stale time for the query client is also 10 minutes